lib/elixir/test/elixir/io/ansi/docs_test.exs

```diff
@@ -57,6 +57,69 @@ defmodule IO.ANSI.DocsTest do
     assert result == "\e[33m### wibble\e[0m\n\e[0m\ntext\n\e[0m"
   end
 
+  test "short single-line quote block is converted into single-line quote" do
```

Comment:
Just one last point: maybe the tests would be clearer with heredocs?

Reply:
Done in 497c29c